### ISSUE TITLE
Scrollbar inconsistently handles clicking on scrollbar region (not thumb)

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/microsoft-ui-xaml#10122]
### Describe the bug

From https://github.com/microsoft/terminal/issues/18063
> Typically when mouse click the scrollbar(not the thumb), hold the mouse button down. The scrollbar will:
page up, , page up, page up, page up, page up,...
Then thumb will stop at the mouse pointer position.
> Instead, there are 3 random behaviors.
> 1. Page up/down once and stop.
> 2. Thumb not stop. And reach the top/bottom position.
> 3. Thumb move across the mouse pointer position.

https://github.com/user-attachments/assets/12989340-50c2-4ff1-a14c-82d142296614

### Steps to reproduce the bug

1. Go to a page with a long scroll region in the WinUI 2 gallery (I used ListView)
2. Click and hold on the scrollbar region (not the thumb)

### Expected behavior

We should be consistent. I'd expect it to scroll to the pointer position.

### NuGet package version

None

**Comment by karkarl** [comment]
Does this bug also repro in WinUI3? This may not meet the bar for WinUI2 servicing.

**Comment by carlos-zamora** [comment]
Yup! Just tried it in WinUI 3 controls gallery (All samples, then made the window very narrow).
